frontend/src/components/dashboard/components/categories-list/categories-list.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, IconName } from 'common/types/types';
+
+import { Category } from './components/category/category';
+import styles from './styles.module.scss';
+
+type Category = {
+  id: number;
+  key: string;
+  name: string;
+};
+
+type Props = {
+  items: Category[];
+};
+
+const CategoriesList: FC<Props> = ({ items }: Props) => {
```

Comment:
```suggestion
const CategoriesList: FC<Props> = ({ items }) => {
```
`FC<Props>` is enough